code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -426,6 +428,22 @@
 			mothdust += 10;
 		if(istype(dna.species, /datum/species/moth))
 			M.mothdust += 10; // End WS edit
+
+	if(M.zone_selected == BODY_ZONE_PRECISE_MOUTH) // Nose boops!
+		nosound = TRUE
+		playsound(src, 'sound/effects/Nose_boop.ogg', 50, 0)
```

Comment:
This sound is jarring and pretty out of place. I'd just cut it.

Reply:
this is fine, trust me

Reply:
I really do not agree

Reply:
ill find another sound effect